src/test/resources/pages/InstructorEditStudentFeedbackPageModified.html

```diff
@@ -119,9 +119,20 @@
               </span>
             </div>
             <div class="col-sm-12">
-              <textarea class="form-control" cols="100%" name="responsetext-1-0" rows="4">
+              <textarea class="form-control" cols="100%" id="responsetext-1-0" name="responsetext-1-0" onkeydown="updateTextQuestionCharCount('responsetext-1-0', 'responseLength-1-0')" onkeyup="updateTextQuestionCharCount('responsetext-1-0', 'responseLength-1-0')" rows="4">
                 Good design
               </textarea>
+              <div class="pull-right margin-top-7px" style="display: ${recommendedLengthDisplay}">
```

Comment:
Anyway, we should avoid inline css styles. This should be handled through css classes and stylesheets.